tools/samtools.py

```diff
@@ -10,6 +10,9 @@
     Current bug with pysam 0.8.1: nosetests does not work unless you use --nocapture.
     python -m unittest works. Something about redirecting stdout.
     Actually, Travis CI still has issues with pysam and stdout even with --nocapture.
+
+    pysam 0.9.1 stops redirecting stdout which makes things much easier,
+    but it's a pain to pip install.
```

Comment:
Fortunately we can install pysam==0.9.1 from [bioconda](https://anaconda.org/bioconda/pysam) where it is the most recent version